### DISCUSSION TITLE
Is there a startup script for ish

### DISCUSSION THREAD

**Discussion by ghost** [ish-app/ish#1633] [General]
Is there a startup script for ish like.ishrc?
TIA, Matt

**Comment by ghost** [comment]
Not sure but since Alpine uses Ash for the shell, maybe `.ashrc`.
> Side note: I haven't try it yet so it might not work.

# Edit
Edit: Tried `.ashrc`. Didn't worked

**Comment by tbodt** [comment]
The startup file for the busybox shell is .profile. HTH
